lib/Service/AppsService.php

```diff
@@ -102,4 +129,17 @@ public function getSpec(string $app): string {
 
 		return json_encode($data);
 	}
+
+	private function getPreview(string $app, string $appPath) : ?string {
```

Comment:
Please use `IAppManager::getAppIcon()` instead.

Reply:
It seems getAppIcon is NC 29, and ocs_api_viewer tells it's NC26 compatible. Do you confirm the NC29 min version ?

Reply:
Oh ok, can you check if there is an alternative API that is usable on 26-31?

Reply:
Otherwise we can just bump the min version to 28, all other versions are already EOL.

Reply:
I didn't find any alternative.

We would need to bump to 29, as it is not available in NC28.

Note : the main (only ?) place in NC core code where this test was done (OC_App::listAllApps()) is still not using getAppIcon() ;)

Reply:
Done.